src/components/Shifting/ShiftDetailsUpdate.tsx

```diff
@@ -331,29 +339,40 @@ export const ShiftDetailsUpdate = (props: patientShiftProps) => {
       />
       <Card className="mx-auto mt-4 w-full max-w-4xl md:p-6 lg:p-8">
         <div className="grid grid-cols-1 gap-4 md:grid-cols-2">
-          <SelectFormField
-            name="status"
-            label={t("status")}
-            required
-            options={
-              careConfig.wartimeShifting
-                ? SHIFTING_CHOICES_WARTIME
-                : SHIFTING_CHOICES_PEACETIME
-            }
-            value={state.form.status}
-            optionLabel={(option) => option.text}
-            optionValue={(option) => option.text}
-            optionDisabled={(option) =>
-              // disable all options except `Destination Approved` for non-admin destination facility users
-              home_facility === state.form.assigned_facility_object?.id &&
-              USER_TYPES.findIndex((type) => user_type === type) <
-                USER_TYPES.findIndex((type) => type === "DistrictAdmin") &&
-              !["DESTINATION APPROVED"].includes(option.text)
-            }
-            optionSelectedLabel={(option) => option.text}
-            onChange={handleFormFieldChange}
-            className="w-full bg-white md:col-span-1 md:leading-5"
-          />
+          <div className="col-span-2 grid grid-cols-1 gap-4 md:grid-cols-2">
```

Comment:
Check the cancel/submit button placement and compare to figma.

Overall, do the same for other elements and try to match.